frontend/src/modules/dataset/dataset-browser.reducer.tsx

```diff
@@ -171,7 +162,7 @@ export const getDatasetContents = createAsyncThunk(
             searchParams.set('prefix', datasetContext.location);
         }
 
-        return axios.get<ListFilesResponse>(`/dataset/${scope}/${datasetContext.name}/files?${searchParams.toString()}`).then((response) => {
+        return axios.get<ListFilesResponse>(`/dataset/${datasetContext.type}/${datasetContext.scope}/${datasetContext.name}/files?${searchParams.toString()}`).then((response) => {
```

Comment:
Going to revert this too - it's going to be broken for now since the API is expecting scope and type in the param, but I'll address this is in the follow-up PR